engine/common/plugin.c

```diff
@@ -2258,13 +2263,18 @@ static void *QDECL PlugBI_GetEngineInterface(const char *interfacename, size_t s
 			Plug_S_PrecacheSound,
 			S_StartSound,
 			S_GetChannelLevel,
+#ifdef VOICECHAT
```

Comment:
Not sure whats best here, perhaps just ifdef this whole `strcmp(interfacename, plugaudiofuncs_name)` block.